### DISCUSSION TITLE
Dollar sign in quotations

### DISCUSSION THREAD

**Discussion by tristone13th** [KaTeX/KaTeX#3431] [Q&A] [answered]
KaTeX will recognize dollar signs in quotations as inline math, but what I want is just two dollar signs to display, how to fix this?

For an exmaple:

> [me@linuxbox ~]\$ echo \$(cal)

In KaTeX, the dollar signs will disappear and the "echo" becomes italic.

For a reference, MathJax has the option `processEscapes: true`  to enable this feature, see [javascript - How to use "$" in non Math context while using MathJax in a Webpage - Stack Overflow](https://stackoverflow.com/questions/54366977/how-to-use-in-non-math-context-while-using-mathjax-in-a-webpage)

**Comment by edemaine** [comment, marked as answer]
One option would be to add `blockquote` (or whatever your quotes are inside) to the `ignoredTags` [auto-render option ](https://katex.org/docs/autorender.html). Or you could use classes to distinguish.

Are you using Markdown to produce your content? If so, a KaTeX plugin for your content would probably help (but only if the quoted text is wrapped in `` ` `` (back ticks)).

It's true that auto-render does not support a `processEscapes` option. I don't think it would be too hard to add one, but it doesn't currently fit our `delimiters` specification, so we'd at least need to figure out what the right API is.

**Reply by tristone13th** [reply]
It really works, thanks for your detailed reply!
